urbit/docker-compose.yml

```diff
@@ -4,8 +4,9 @@ services:
     environment:
       APP_HOST: urbit_manager_1
       APP_PORT: 8090
+    PROXY_AUTH_ADD: "false"
   
-  manager:
+    image: mopfelwinrux/urbit-umbrel:v2.0@sha256:5c628df5007e3c3e8320b6e3f2e97a120e5a448ab5821bbe5fb21c922d64d3e4
     image: mopfelwinrux/urbit-umbrel:v1.22@sha256:fd6303520a4ef3dec9be223b099af6a2299810a277b921e574e135a96a2eb39f
```

Comment:
The `manager` service heading was accidentally deleted and now there are two images (new v2.0 and v.1.22).

Reply:
Woops! Adding the new env var line broke the parser in my update script and I missed this. Fixed now